src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: Avenir-Heavy;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.labelSearch {
+  box-sizing: border-box;
```

Comment:
it's better to apply `box-sizing` for all elements and move it to `*` selector
`* { box-sizing: border-box; }`